src/libmongoc/doc/mongoc_find_and_modify_opts_t.rst

```diff
@@ -10,9 +10,9 @@ Synopsis
 
 ``mongoc_find_and_modify_opts_t`` is a builder interface to construct a `find_and_modify <https://www.mongodb.com/docs/manual/reference/command/findAndModify/>`_ command.
 
-It was created to be able to accommodate new arguments to the MongoDB find_and_modify command.
+It was created to be able to accommodate new arguments to the `MongoDB findAndModify command <https://www.mongodb.com/docs/manual/reference/command/findAndModify/>`_.
```

Comment:
I recommend de-duping this link and adding something to the `rst_prolog` in `src/libmongoc/doc/conf.py`:

```rst
.. _the MongoDB findAndModify command:
    https://www.mongodb.com/docs/manual/reference/command/findAndModify/
```

and then use it as:

```suggestion
It was created to be able to accommodate new arguments to `the MongoDB findAndModify command`_.
```

this will check the spelling at each use (since it must be spelled equivalent to the link target) and give a central location to put and update such links.

Reply:
I like it. Thank you for the suggestion.